commit cedab54edb73976a382aa23c7b41f5403f4e2352
Author: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date:   Tue Dec 27 16:02:36 2016 -0200

    am437x-vpfe: always assign bpp variable
    
    commit 6ebf75774f823ddbdbd10921006989d4df222f4a upstream.
    
    In vpfe_s_fmt(), when the sensor format and the requested format were
    the same, bpp was assigned to vpfe->bpp without being initialized first.
    
    Grab the bpp value that is currently used by using __vpfe_get_format()
    instead of its wrapper, vpfe_try_fmt().
    
    This use of uninitialized variable has been found by compiling the
    kernel with clang.
    
    Fixes: 417d2e507edc ("[media] media: platform: add VPFE capture driver
    support for AM437X")
    
    Signed-off-by: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
    Signed-off-by: Hans Verkuil <hans.verkuil@cisco.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1221_cedab54edb73976a382aa23c7b41f5403f4e2352.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1221_cedab54edb73976a382aa23c7b41f5403f4e2352.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
